syz-fuzzer/fuzzer_test.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2019 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package main
+
+import (
+	"fmt"
+	"math"
+	"math/rand"
+	"testing"
+
+	"github.com/google/syzkaller/prog"
+)
+
+func TestChooseProgramFromCorpus(t *testing.T) {
+	r := rand.New(rand.NewSource(0))
+	target, err := prog.GetTarget("test", "64")
+	if err != nil {
+		t.Fatal(err)
+	}
+	fuzzer := &Fuzzer{}
+
+	const (
+		maxIters   = 100000
+		maxPrio    = 2000000
+		sizeCorpus = 100000
+		eps        = 0.01
+	)
+
+	var priorities []int64
+	for i := 0; i < sizeCorpus; i++ {
+		p, err := target.Deserialize([]byte(fmt.Sprintf("mutate9(0x%x)", i)), prog.NonStrict)
+		if err != nil {
+			t.Fatalf("failed to deserialize the program: %v", err)
+		}
+		fuzzer.corpus = append(fuzzer.corpus, p)
+		prio := r.Int63n(maxPrio)
+		priorities = append(priorities, prio)
+		fuzzer.sumPrios += prio
+		fuzzer.corpusPrios = append(fuzzer.corpusPrios, fuzzer.sumPrios)
```

Comment:
Here you are duplicating production code, so you are both adding more code, testing code that is not relevant and not testing code that actually runs in production. Try to find a way to use/test addInputToCorpus.